client/source/components/RR/tests/ReviewList.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen, cleanup } from '@testing-library/react';
+import { reviews } from './RRdata.js';
```

Comment:
You are destructuring here, but export a single default from `RRData`

Reply:
Also, this test currently does not pass.

Reply:
Updated the method with .toContain and added jest environment up top